.github/workflows/PR.yml

```diff
@@ -8,18 +8,19 @@ jobs:
     runs-on: ubuntu-18.04
 
     steps:
-    - uses: actions/checkout@v2
+    - uses: actions/checkout@v3
 
     - name: Validate Gradle Wrapper
       uses: gradle/wrapper-validation-action@v1
 
-    - name: Cache gradle
-      uses: actions/cache@v2.1.0
+    - name: Set up JDK 11
+      uses: actions/setup-java@v3.1.1
       with:
-        path: ~/.gradle/caches
-        key: ${{ runner.os }}-gradle-${{ hashFiles('**/*.gradle*') }}
-        restore-keys: |
-          ${{ runner.os }}-gradle-
+        distribution: 'zulu'
+        java-version: 11
```

Comment:
doesnt this need to be 17?

Reply:
I doesn't need to be 17 because I use grammarkit composer 0.1.10, because EE only use 2022.1 so I though we don't need to use intellij-grammar 2022.3, which requires 17 (and grammarkit compose 0.1.11)